package.json

```diff
@@ -21,7 +21,7 @@
         "test": "yarn test:dev",
         "test:dev": "NODE_ENV=dev yarn workspace @zkmpc/backend emulator:exec-test",
         "test:prod": "export GOOGLE_APPLICATION_CREDENTIALS=\"./packages/backend/serviceAccountKey.json\" && NODE_ENV=prod yarn workspace @zkmpc/actions test:contracts && NODE_ENV=prod jest --config=jest.json --detectOpenHandles --forceExit --runInBand --coverage",
-        "test:ci-prod": "NODE_ENV=prod jest --config=jest.json --detectOpenHandles --forceExit --runInBand --coverage=false",
+        "test:ci-prod": "export GOOGLE_APPLICATION_CREDENTIALS=\"./packages/backend/serviceAccountKey.json\" && NODE_ENV=prod jest --config=jest.json --detectOpenHandles --forceExit --runInBand --coverage=false",
```

Comment:
Hmm why did you add the export here? When we run the tests in CI, the environment variables are used to specify the `serviceAccountKey.json` credential too. Am I missing something new there? Correct me if I'm wrong ^^

Reply:
If you are sure that this path is also valid in the IC, that is fine with me. Much neater and more readable!